pom.xml

```diff
@@ -97,5 +105,25 @@
             <artifactId>calculator-app</artifactId>
             <version>${calculator-app.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>${selenium-java.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>${webdrivermanager.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.0-alpha2</version>
```

Comment:
1. Please move version to the `properties` section
2. It will be better use stable release of library instead of alpha

Reply:
I have changed versions on stable and moved it to the properties section.